commit 4606b98145e1adfd2124f86204aada408d5d1c20
Author: Oleg Gashev <oleg@gashev.net>
Date:   Tue May 7 16:04:30 2013 -0400

    Fixed timestamp regexp to t/dists.t file.
    
    If time zone is EDT, failed tests from t/dists.t file: Generated tree matches
    template. Problem with incorrect timestamp regexp.

diff --git a/t/dists.t b/t/dists.t
index f13e477..6bc4443 100755
--- a/t/dists.t
+++ b/t/dists.t
@@ -69,9 +69,9 @@ sub compare_tree {
                         # likewise, it is normal that the timestamp in the changelog differs
                         return 0
                             if $a
-                                =~ /^ -- Joe Maintainer <joemaint\@test\.local>  \w+, \d+ \w+ \d+ \d+:\d+:\d+ \+\d+$/
+                                =~ /^ -- Joe Maintainer <joemaint\@test\.local>  \w+, \d+ \w+ \d+ \d+:\d+:\d+ (\+|-)\d+$/
                                 and $b
-                                =~ /^ -- Joe Maintainer <joemaint\@test\.local>  \w+, \d+ \w+ \d+ \d+:\d+:\d+ \+\d+$/;
+                                =~ /^ -- Joe Maintainer <joemaint\@test\.local>  \w+, \d+ \w+ \d+ \d+:\d+:\d+ (\+|-)\d+$/;
 
                         return $a ne $b;
                     }
